src/ui/inspector/InspectorViews.vue

```diff
@@ -21,7 +21,7 @@
  *****************************************************************************/
 
 <template>
-<div></div>
+<div class="js-inspector-views"></div>
```

Comment:
If this is solely used for e2e, we should consider using an `aria-label` instead

Reply:
Hmm There is nothing here to label though. It represents a section. Adding a label doesn't make sense?

Reply:
It's an accessible label, not a visible label. It will be used for a11y screen readers and for selectors in our e2e tests